sda-commons-server-openapi-example/src/main/java/org/sdase/commons/server/openapi/example/people/rest/PersonService.java

```diff
@@ -0,0 +1,102 @@
+package org.sdase.commons.server.openapi.example.people.rest;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.headers.Header;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.security.SecurityRequirements;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+import org.sdase.commons.shared.api.error.ApiError;
+
+// Identify this interface for Swagger documentation.
+//
+// We prefer to put these annotations into a separate interface to make the code of the endpoint
+// more readable because it contains less boilerplate code.
+//
+// Requires both a Authorization and a Consumer-Token. All requests inside are tagged as "People",
+// tags help to group the documentation for better navigation.
+@SecurityRequirements({
+  @SecurityRequirement(name = AuthDefinition.BEARER_TOKEN),
+  @SecurityRequirement(name = AuthDefinition.CONSUMER_TOKEN)
+})
+@Tag(name = "People")
+@Path("people") // define the base path of this endpoint
+@Produces({
+  APPLICATION_JSON,
+  "application/hal+json"
+}) // should be set to produce 406 for other accept headers
+@Consumes({APPLICATION_JSON}) // should be set to produce 406 for other accept headers
```

Comment:
I hope wrong Content-type results in 415 and not 406
```suggestion
@Consumes({APPLICATION_JSON}) // should be set to produce 415 for other Content-type headers
```

Does the generator ignore this for GET requests or should it be annotated at method level?